templates/github-webhooks.yaml

```diff
@@ -0,0 +1,9 @@
+{{- range .Values.github_webhook }}
+---
+apiVersion: camptocamp.com/v1
+kind: GitHubWebhook
+metadata:
+  name: {{ include "common.fullname" ( dict "root" $ "service" $.Values ) }}-{{ .name }}
+  labels: {{ include "common.labels" ( dict "root" $ "service" $.Values ) | nindent 4 }}
+spec: {{- toYaml .content | nindent 2 }}
```

Comment:
just seeing the difference between GithubWebhook an SharedConfigSource, I think that for consistency `spec` should be based on the one from SharedConfigSource, i.e.  adding `content`and `name` . 
What do you think @sbrunner ?

Reply:
=> ok (by meet :-) )